provider-middleware/brightspace.go

```diff
@@ -17,12 +17,13 @@ import (
 )
 
 const (
-	CsvDownloadPath     = "csvs"
 	TokenEndpoint       = "https://auth.brightspace.com/core/connect/token"
-	DataSetsEndpoint    = "https://unlocked.brightspacedemo.com/d2l/api/lp/1.28/dataExport/bds/list"
-	DataDownloadEnpoint = "https://unlocked.brightspacedemo.com/d2l/api/lp/1.28/dataExport/bds/download/%s"
+	DataSetsEndpoint    = "https://unlocked.brightspacedemo.com/d2l/api/lp/%s/dataExport/bds/list"
+	DataDownloadEnpoint = "https://unlocked.brightspacedemo.com/d2l/api/lp/%s/dataExport/bds/download/%s"
 )
 
+var CsvDownloadPath = os.Getenv("BRIGHTSPACE_TEMP_DIR")
```

Comment:
Can this get a default value in the `docker compose` file? also I will need to remember to set this value in the helm chart

Reply:
I included this in the docker-compose.yml file, unless I put in the wrong location? I placed this value within the provider-service image configuration.

![image](https://github.com/user-attachments/assets/90b1452d-8145-4566-a694-f4463c81eaaa)

Reply:
I must have missed that i swore i looked

Reply:
i was thinking I may have put in the wrong place. :)